### PR TITLE
Add warning if configuration file was not found

### PR DESCRIPTION
**Pull request by abaranch**

### CODE REVIEW

File: src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs [hunk 1]
```diff
@@ -279,6 +279,18 @@ public void ComponentInitializationConfigurationError(string type, string error,
                 this.nameProvider.Name);
         }
 
+        [Event(
```
Comment (iusafaro) [inline src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs]: should we mark it as UserActionable?

**Comment by iusafaro** [comment]
:shipit:

File: src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs [hunk 2]
```diff
@@ -279,6 +279,18 @@ public void ComponentInitializationConfigurationError(string type, string error,
...
+            23,
+            Message = "ApplicationInsights configuration file '{0}' was not found.",
+            Level = EventLevel.Warning)]
+        public void ApplicationInsightsConfigNotFoundWarning(string directory, string appDomainName = "Incorrect")
```
Comment (dnduffy) [inline src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs]: nitpick: Is it really "directory" or is it "filename"?

**Comment by dnduffy** [comment]
:shipit:

File: src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs (hunk 2 above)
Comment (abaranch) [inline src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs]: Resolved with next checkin. Thank you.

---
In reply to: [61342031](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/224#discussion_r61342031) [](ancestors = 61342031)

**Pull request merged**